### PR TITLE
Add API documentation to '@theia/preview'

### PR DESCRIPTION
**Pull request by sdirix**
* Document preview-handler.ts
* Add extension guide to README.md

Signed-off-by: Stefan Dirix <sdirix@eclipsesource.com>
Contributed on behalf of STMicroelectronics

### CODE REVIEW

File: packages/preview/src/browser/preview-handler.ts [hunk 1]
```diff
@@ -48,13 +109,19 @@ export class PreviewHandlerProvider {
...
+     */
     findContribution(uri: URI): PreviewHandler[] {
         const prioritized = Prioritizeable.prioritizeAllSync(this.previewHandlerContributions.getContributions(), contrib =>
             contrib.canHandle(uri)
         );
         return prioritized.map(c => c.value);
     }
 
+    /**
+     * Indicates whether any PreviewHandler wants to take on the resource identified by the given URI.
```
Comment (tortmayr) [inline packages/preview/src/browser/preview-handler.ts]: Maybe it's just me but "wants to take on" sounds a bit weird in that context.

**Review by tortmayr — APPROVED** [review]
Looks good to me!
(I fixed a minor typo but other than that everything is fine)

**Comment by sdirix** [comment]
Thanks! I updated the header to the new format and squashed the commits.

**Comment by sdirix** [comment]
Reverted the copyright header changes

**Comment by sdirix** [comment]
Closed in favor of #15

**Pull request closed without merging**